.vscode/launch.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
Editor specific settings should generally be in global gitignore

Reply:
very true. just offering it in case someone doesn't want to figure it out on their own.